policies/Sponsorship_Policy.md

```diff
@@ -0,0 +1,154 @@
+
+Where a sponsorship is reported to the OMC as a potential inappropriate sponsor
+the above criteria will be used to assess that sponsorship regardless of where
+the sponsorship resides, i.e. [GitHub Sponsors] or direct arrangement with 
+OpenSSL.
+
+
+**Recording Sponsorship of OpenSSL**
+
+* For Sponsorship made via [GitHub Sponsors], <$400/month and one-off payments,  
+the information provided via [GitHub Sponsors] registration process is deemed 
+sufficient record of the sponsorship provided.
+* For Sponsorship at Platinum Level or above, a Sponsorship Agreement, should 
+be entered into. It should clearly set out:
+   * the rights and obligations between both parties;
+   * clearly confirm any commitments made to each other;
+   * include Term of Sponsorship, any exclusivity requirements, payment 
+requirements, agreed benefits, and termination of agreement details.
+* For all Sponsorship, at Bronze Level or above, (including GitHub Sponsors 
+$400/month or above) the sponsorship must be documented and recorded in an 
+appropriate system so that it can be identified and managed.
+
+Details to be recorded in this system are:
+     * Name of organisation, individual or entity providing sponsorship;
+     * Duration of sponsorship arrangement, including start and end dates (where applicable);
+     * Value of sponsorship in $US and local currency (where applicable);
+     * Details of what is being supplied under the sponsorship e.g. financial aid, 
+resources etc;
+     * Any conditions that apply to the sponsorship;
+     * Whether permission to display logos, company details and links to their 
+websites has been provided;
+     * Contact details for the organisation, Individual or Entity.
+
+* Acknowledgement of Sponsors on OpenSSL's website, including logos, company 
+details and links to external websites will only ocurr with express permission 
+of the sponsoring organisation, individual or entity.  At the end of the 
+sponsorship period, where sponsorship is not renewed, acknowledgements will be 
+removed from the OpenSSL website.
+
+* For sponsors on [GitHub Sponsors] that have elected to be _public_, it is 
+deemed that they have provided express permission for such acknowledgement as 
+provided by [GitHub Sponsors] and to receiving direct contact from OpenSSL.
+
+
+#### OpenSSL Providing Sponsorship
+
+
+Below sets out how OpenSSL will assess requests to provide sponsorship.
+
+The OMC will assess each application for sponsorship, under the following 
+criteria:
+* Does sponsoring support OpenSSL's Mission and values, it's goals, objectives and priorities;
+* Could sponsoring be perceived as a conflict of interest, or impartial;
+* Does sponsoring breach any legal or regulatory requirements;
+* Will sponsoring damage OpenSSL's reputation;
+* What is the financial implication of sponsoring to OpenSSL;
+* What are the benefits of sponsoring this organisation, individual or entity;
+* Does sponsoring further Open Source principles.
```

Comment:
Isn't this covered by the mission and values?

Reply:
The M&V are at a higher level, the sponsorship needs to align with them but, for the policy I thought we'd call out some more specific things like the legal requirements, financial impacts which are implied in the M&V but not specifically mentioned.